tests/unit/test_status.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+
+import pytest
+
+from cromshell.status import command as status_command
+
+
+class TestStatus:
+    """Test the status command functions"""
+
+    def test_workflow_failed_doom(self, mock_data_path):
+        workflow_metadata_path = os.path.join(
```

Comment:
failed and doom are two different statuses to me, so maybe we could name it differently

Reply:
Be more descriptive with the title of the function, example test_workflow_failed_metadata_doom. Try looking at pythonic suggestion or look at gatk test functions.

Reply:
done, 
A quick search shows the common convention is to  "start with test + name of the code/unit separated by an underscore"